tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -61,7 +66,12 @@ private string GetStatusString(string id)
     public QuietBackgroundProcessesViewModel(IExperimentationService experimentationService)
     {
         _experimentationService = experimentationService;
+        _sessionStateText = string.Empty;
+
         IsFeaturePresent = QuietBackgroundProcessesSessionManager.IsFeaturePresent();
```

Comment:
>QuietBackgroundProcessesSessionManager [](http://example.com/codeflow?start=27&length=38)

turns out this ViewModel constructor is expensive enough (loading QuietBackgroundProcessesSessionManager, etc., that is causes noticeable delays when navigating to Customization. 

I have a branch here that offloads to a background thread - [https://github.com/microsoft/devhome/commit/26e48cc14a72e71160a73da3d47f3d598cf1c7bd](https://github.com/microsoft/devhome/commit/26e48cc14a72e71160a73da3d47f3d598cf1c7bd).

I'm wondering if it may be better to incorporate that in this change, or if I should go ahead with that review and we handle the merge?

Reply:
Definitely commit that and we'll deal with any gross merging later.  I knew there was a delay, but I assumed it was just on first run when Defender gets nosy when we're launching a freshly-built OOP exe.

Reply:
Cool, here's the PR for quiet mode - [https://github.com/microsoft/devhome/pull/2621](https://github.com/microsoft/devhome/pull/2621)

Reply:
This is merged to main now so feel free to sync up those changes